library/build.gradle

```diff
@@ -1,8 +1,17 @@
 plugins {
```

Comment:
Should `gradle-wrapper.jar` be uploaded here?

Reply:
gradle-wrapper.jar is needed to download gradle-wrapper for the project folder